internal/cms/controller/cms_controller.go

```diff
@@ -0,0 +1,93 @@
+package controller
+
+import (
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/labstack/echo/v4"
+	"github.com/wanrun-develop/wanrun/internal/cms/core/dto"
+	"github.com/wanrun-develop/wanrun/internal/cms/core/handler"
+	"github.com/wanrun-develop/wanrun/internal/wrcontext"
+	"github.com/wanrun-develop/wanrun/pkg/errors"
+	"github.com/wanrun-develop/wanrun/pkg/log"
+)
+
+type ICmsController interface {
+	UploadFile(c echo.Context) error
+}
+
+type cmsController struct {
+	ch handler.ICmsHandler
+}
+
+func NewCmsController(ch handler.ICmsHandler) ICmsController {
+	return &cmsController{ch}
+}
+
+func (cc *cmsController) UploadFile(c echo.Context) error {
+	logger := log.GetLogger(c).Sugar()
+
+	// claims情報の取得
+	claims, wrErr := wrcontext.GetVerifiedClaims(c)
+
+	if wrErr != nil {
+		return wrErr
+	}
+
+	// dogOwnerIDの取得
+	dogOwnerID, wrErr := claims.GetDogOwnerIDAsInt64(c)
+	if wrErr != nil {
+		return wrErr
+	}
```

Comment:
このログインユーザーのIDをとるのは、wrcontextから取る１つの関数を作りたい
→wrcontextのこの関数を取れば、int64のユーザーIDをみんな共通的にとれる関数をひとつ作っておくと人によって取り方変わらなくて、良き！
```suggestion
	// claims情報の取得
	dogOwnerID, wrErr := wrcontext.GetLoginUserID(c)
	if wrErr != nil {
		return wrErr
	}
```